static/scss/_button.scss

```diff
@@ -49,6 +49,19 @@
     border-radius: 3px;
     white-space: nowrap;
 
+    &:hover,
```

Comment:
@marc-farre Small question, is this related to the tour?

Reply:
No. Anyway, I have rewritten completely this _button.scss: https://github.com/humhub/humhub-internal/issues/661